README_plasma.txt

```diff
@@ -0,0 +1,33 @@
+University of Illinois at Urbana-Champaign - September 2015
+
+In this branch, Cantera has been extended to Plasma flows.
+
+New capabilities:
```

Comment:
Documentation should be added as Doxygen docstrings in the header files and in the Sphinx documentation, both of which are used to produce the online documentation.